packages/store-indexer/src/postgres/createQueryAdapter.ts

```diff
@@ -13,11 +14,11 @@ import { getAddress } from "viem";
  */
 export async function createQueryAdapter(database: PgDatabase<any>): Promise<QueryAdapter> {
   const adapter: QueryAdapter = {
-    async findAll(chainId, address) {
-      const internalTables = buildInternalTables();
-      const tables = (await getTables(database)).filter(
-        (table) => address != null && getAddress(address) === getAddress(table.address)
-      );
+    async findAll({ chainId, address, tableIds = [] }) {
+      const includedTableIds = new Set(tableIds.length ? [...internalTableIds, ...tableIds] : []);
+      const tables = (await getTables(database))
+        .filter((table) => address == null || getAddress(address) === getAddress(table.address))
+        .filter((table) => !includedTableIds.size || includedTableIds.has(table.tableId));
```

Comment:
general q, not related to this PR: should we combine this adapter with the one defined in `sqlite`? 

On the same note, a couple lines below this we have `const sqliteTable = builtTable(table)`, should that be `postgresTable`? Is it the same object type? (Can't comment directly on it bc it didn't change in this PR)

Reply:
I would like to, but there's some API differences between drizzle's postgres and sqlite adapters that makes it hard to combine these.

Also the sqlite patterns here are a bit behind - I cleaned things up in the postgres, but didn't go back to refactor sqlite to match.